include/nanogui/textbox.h

```diff
@@ -66,8 +66,10 @@ class NANOGUI_EXPORT TextBox : public Widget {
     /// Set the \ref Theme used to draw this widget
     virtual void setTheme(Theme *theme) override;
 
-    /// Set the change callback
+    /// The callback to execute when the value of this TextBox has changed.
```

Comment:
The rest of `textbox.h` is still a TODO.  I only updated things to make docs link back and forth.  Same happened in `widget.h`.